docs/en/latest/plugins/authz-keycloak.md

```diff
@@ -42,9 +42,9 @@ For more information on Keycloak, refer to [Keycloak Authorization Docs](https:/
 
 | Name                           | Type          | Requirement | Default                                       | Valid                                                              | Description                                                                                                                                                 |
 | ------------------------------ | ------------- | ----------- | --------------------------------------------- | ------------------------------------------------------------------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| discovery                      | string        | optional    |                                               | https://host.domain/auth/realms/foo/.well-known/uma2-configuration | URL to discovery document for Keycloak Authorization Services.                                                                                              |
-| token_endpoint                 | string        | optional    |                                               | https://host.domain/auth/realms/foo/protocol/openid-connect/token  | A OAuth2-compliant Token Endpoint that supports the `urn:ietf:params:oauth:grant-type:uma-ticket` grant type. Overrides value from discovery, if given.     |
-| resource_registration_endpoint | string        | optional    |                                               | https://host.domain/auth/realms/foo/authz/protection/resource_set  | A Keycloak Protection API-compliant resource registration endpoint. Overrides value from discovery, if given.                                               |
+| discovery                      | string        | optional    |                                               | `https://host.domain/auth/realms/foo/.well-known/uma2-configuration` | URL to discovery document for Keycloak Authorization Services.                                                                                              |
```

Comment:
Why add '``' surround the link?

Reply:
I thought they are just **valid url format** and it seems they don't direct to anywhere.

![2](https://user-images.githubusercontent.com/53715187/116960456-0c983f00-acbe-11eb-9928-24af16ae6f53.JPG)
![1](https://user-images.githubusercontent.com/53715187/116960447-073af480-acbe-11eb-9001-a413888aeb15.JPG)

Reply:
I can't understand what you mean. Does adding '``' make things different?

Reply:
I thought ,If a phrase doesn't  direct to any page ,we could better go with `` rather than explicitly showing it as a link,,something like 
![100](https://user-images.githubusercontent.com/53715187/116969453-e8dff380-acd3-11eb-9b79-f9f287322275.JPG)
![101](https://user-images.githubusercontent.com/53715187/116969459-eaa9b700-acd3-11eb-9d0e-c8863a36f7a8.JPG)

It would be more helpful  to me if you @spacewander  explain me why you used (``) `http://127.0.0.1:9080/apisix/dashboard/` in https://apisix.apache.org/docs/apisix/plugins/limit-req#how-to-enable-on-the-route-or-serivce. As I really don't know how to explain this, I would like to request @juzhiyuan too to review this PR ;)

Reply:
> It would be more helpful to me if you @spacewander explain me why you used (``) http://127.0.0.1:9080/apisix/dashboard/ in apisix.apache.org/docs/apisix/plugins/limit-req#how-to-enable-on-the-route-or-serivce.

You should not ask me. If you know how to use `git blame`, you can see **every lines in this page is not written by me**. How can I answer you?